jenkins/PerfCI-Chaos/04-PerfCI-Chaos-Upgrade-OpenShift-Deployment /Jenkinsfile

```diff
@@ -0,0 +1,296 @@
+                                        -e BUILD_VERSION='${build_version}' \
+                                        -e RUN_TYPE='${RUN_TYPE}' \
+                                        -e SAVE_ARTIFACTS_LOCAL=${SAVE_ARTIFACTS_LOCAL} \
+                                        -e ENABLE_PROMETHEUS_SNAPSHOT=${ENABLE_PROMETHEUS_SNAPSHOT} \
+                                        -e WORKER_DISK_IDS='${WORKER_DISK_IDS}' \
+                                        -e WORKER_DISK_PREFIX='${WORKER_DISK_PREFIX}' \
+                                        -e SCALE='${SCALE}' \
+                                        -e SCALE_NODES='${SCALE_NODES}' \
+                                        -e REDIS='${REDIS}' \
+                                        -e THREADS_LIMIT='${THREADS_LIMIT}' \
+                                        -e WINDOWS_URL='${WINDOWS_URL}'  \
+                                        -e DELETE_ALL=${DELETE_ALL} \
+                                        -e VERIFICATION_ONLY=${VERIFICATION_ONLY} \
+                                        -e LSO_NODE='${LSO_NODE}'  \
+                                        -e TIMEOUT='${TIMEOUT}' \
+                                        -e log_level='INFO' \
+                                        -v '${KUBECONFIG_PATH}:${KUBECONFIG_PATH}' \
+                                        --privileged '${QUAY_BENCHMARK_RUNNER_REPOSITORY}'
+                                    """
+                                    success = true
+                                }
+                            } catch (Exception e) {
+                                echo "Error occurred in workload ${workload}: ${e.message}"
+                                if (i < retryCount - 1) {
+                                    echo "Retrying..."
+                                    sleep(time: 120, unit: 'SECONDS') // Sleep for 2 minutes between retries
+                                } else {
+                                    echo "Reached maximum retry attempts for workload ${workload}. Moving to the next workload..."
+                                }
+                            }
+                        }
+                        if (!success) {
+                            echo "Workload ${workload} failed after ${retryCount} retries."
+                        }
+                        // sleep 10 sec between each VMs validation interation
+                        sleep(time: 10, unit: 'SECONDS')
+                    } // while
+                 }
+                }
+            }
+        } // WORKLOADS Deployment
+    }
+        }
+    } // stages
+
+    post {
+        always {
+            script {
+                sh '''
+                    if [[ "$(sudo podman images -q ${QUAY_BENCHMARK_RUNNER_REPOSITORY} 2> /dev/null)" != "" ]]; then
+                        sudo podman rmi -f $(sudo podman images -q ${QUAY_BENCHMARK_RUNNER_REPOSITORY} 2> /dev/null)
+                    fi
```

Comment:
See above about running containers.

Reply:
The main purpose of this deletion is to remove all running containers of benchmark-runner to allow a fresh deployment.